### PR TITLE
fix: fix typos in documentation

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S ./.yarn,./CHANGELOG -L afterall,alo,aswell,celcius,keypair,mot,nd,nin,parm,selv,sensative,unsecure`

### CODE REVIEW

File: .github/workflows/dependency-review.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Dependency Review Action
 #
-# This Action will scan dependency manifest files that change as part of a Pull Reqest, surfacing known-vulnerable versions of the packages declared or updated in the PR. Once installed, if the workflow run is marked as required, PRs introducing known-vulnerable packages will be blocked from merging.
+# This Action will scan dependency manifest files that change as part of a Pull Request, surfacing known-vulnerable versions of the packages declared or updated in the PR. Once installed, if the workflow run is marked as required, PRs introducing known-vulnerable packages will be blocked from merging.
```
Comment (AlCalzone) [inline .github/workflows/dependency-review.yml:3]: Lol, this came straight from Github

File: docs/api/driver.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ new (port: string, options?: ZWaveOptions) => Driver
 
 The first constructor argument is the address of the serial port. On Windows, this is similar to `"COM3"`. On Linux this has the form `/dev/ttyAMA0` (or similar). Alternatively, you can connect to a serial port that is hosted over TCP (for example with the `ser2net` utility), see [Connect to a hosted serial port over TCP](usage/tcp-connection.md).
 
-For more control, the constructor accepts an optional options object as the second argument. See [`ZWaveOptions`](#ZWaveOptions) for a detailed desription.
+For more control, the constructor accepts an optional options object as the second argument. See [`ZWaveOptions`](#ZWaveOptions) for a detailed desrciption.
```
Comment (AlCalzone) [inline docs/api/driver.md]:
```suggestion
For more control, the constructor accepts an optional options object as the second argument. See [`ZWaveOptions`](#ZWaveOptions) for a detailed description.
```

File: docs/api/node.md [hunk 3]
```diff
@@ -900,7 +900,7 @@ When configuring devices or during longer message exchanges, this behavior may b
 
 ## ZWaveNode events
 
-The `Node` class inherits from the Node.js [EventEmitter](https://nodejs.org/api/events.html#events_class_eventemitter) and thus also supports its methods like `on`, `removeListener`, etc. The available events are avaiable:
+The `Node` class inherits from the Node.js [EventEmitter](https://nodejs.org/api/events.html#events_class_eventemitter) and thus also supports its methods like `on`, `removeListener`, etc. The available events are available:
```
Comment (AlCalzone) [inline docs/api/node.md]:
```suggestion
The `Node` class inherits from the Node.js [EventEmitter](https://nodejs.org/api/events.html#events_class_eventemitter) and thus also supports its methods like `on`, `removeListener`, etc. The following events are available:
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Thanks. Found two more in the changed lines though :)

**Review by AlCalzone — APPROVED** [review]

**Comment by luzpaz** [comment]
@AlCalzone I also found typos in a bunch of `packages/config/config/devices/**/*.json` files. Can I make a PR for them ?

**Comment by AlCalzone** [comment]
Go ahead!

**Pull request merged**